packages/cli/src/ui/AppContainer.tsx

```diff
@@ -1352,12 +1355,26 @@ Logging in with Google... Restarting Gemini CLI to continue.
 
     coreEvents.on(CoreEvent.UserFeedback, handleUserFeedback);
 
+    const handleConsoleLog = (payload: ConsoleLogPayload) => {
+      if (payload.type === 'error') {
```

Comment:
I think we should have some throttle or deduplication mechanism here.If a component enters a render loop or throws errors repeatedly, the chat UI will be flooded with identical error bubbles, making the application unusable.